charts/foundry/templates/deployment.yaml

```diff
@@ -104,7 +104,10 @@ spec:
             - "--balance"
             - {{ .Values.anvil.accountBalance | quote }}
             {{- end }}
-            - "--prune-history"
+            - "--state"
+            - "/data/dump.json"
+            - "--state-interval"
+            - "10"
```

Comment:
If the data dump happens on the intervals and is not in the a realtime, maybe it would be better to push this to S3. How large is the data dump?

Does `10` means, dump data every 10 seconds?

Reply:
yes right ! it dumps every 10s to the file location specified in `--state` args and during start up it loads from that file if the file exists